### PR TITLE
Add Python 3.6 support

### PR DESCRIPTION
**Pull request by sodre**

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge-admin, please rerender

**Comment by sodre** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ outputs:
         - python
         - alembic >=0.8.3,<0.9
         - bleach ==2.1.2
-        - configparser >=3.5.0, <3.6.0
+        - configparser >=3.5.0, <3.6.0  # [py<36]
```
Comment (jakirkham) [inline recipe/meta.yaml]: So think we don't need this on Python 3.5 either as it's a backport package from Python 3.5 to older Python version.

Comment (sodre) [reply recipe/meta.yaml]: I'll try it that way and try to reenable 3.5

**Pull request merged**